session-signatures/getLitActionSessionSigs/src/index.ts

```diff
@@ -48,20 +50,40 @@ export const getSessionSigsLitAction = async () => {
       `✅ Minted new PKP with public key: ${pkp.publicKey} and ETH address: ${pkp.ethAddress}`
     );
 
+    console.log("🔄 Minting Capacity Credits NFT...");
+    const capacityTokenId = (
+      await litContracts.mintCapacityCreditsNFT({
+        requestsPerKilosecond: 10,
+        daysUntilUTCMidnightExpiration: 1,
+      })
+    ).capacityTokenIdStr;
+    console.log(`✅ Minted new Capacity Credit with ID: ${capacityTokenId}`);
```

Comment:
I think the `getSessionSigsLitAction` function should take a Capacity Credit as an argument, and only mint one if one isn't provided - what do you think? There's no need to mint a credit every single time

Reply:
If we do implement this, we should probably do the same for PKPs? We just don't want users thinking they **have** to mint new credits/PKPs

Reply:
Alternatively we could have users provide the `capacityTokenIdStr` in the `.env` file. Thoughts?

Reply:
Well passing it as an argument makes it easier to write the test files. For the tests we want to mint new ones and use existing ones, and that's harder to do if it's read from `.env`